python3/relocate.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# This script ensures that the installation's sysconfig module contains the
+# correct paths. Without this, any C modules will fail to compile.
+
+root="$1"
+cd "${root}"
+
+sysconfig="$(echo install/lib/python*/_sysconfigdata_*.py)"
+tmpfile="$(mktemp)"
+trap "rm -f ${tmpfile}" INT EXIT
```

Comment:
```suggestion
trap 'rm -f ${tmpfile}' INT EXIT
```

```
In relocate.sh line 11:
trap "rm -f ${tmpfile}" INT EXIT
            ^--------^ SC2064 (warning): Use single quotes, otherwise this expands now rather than when signalled.
```

Reply:
TIL - nice one shellcheck

Reply:
shellcheck is nice, but this change is unnecessary, as shellcheck itself says:

> If you don't care that the trap code is expanded early because the commands/variables won't change during execution of the script, or because you want to use the current and not the future values, then you can ignore this message.

Reply:
Conversely, making the "unnecessary" change means that future generations won't trip over this shellcheck error. Also we could add shellcheck to the CI for hermit-packages!